t/ui/06-operator_links.t

```diff
@@ -35,59 +35,48 @@ unless ($driver) {
 my $t = Test::Mojo->new('OpenQA::WebAPI');
 
 # we don't want to test javascript here, so we just test the javascript code
-# List with no login
+note 'List with no login';
 $t->get_ok('/tests')->status_is(200)->content_like(qr/is_operator = false;/, "test list rendered without is_operator");
-
-# List with an authorized user
+note 'List with an authorized user';
 $test_case->login($t, 'percival');
 $t->get_ok('/tests')->status_is(200)->content_like(qr/is_operator = true;/, "test list rendered with is_operator");
-
-# List with a not authorized user
+note 'List with a not authorized user';
 $test_case->login($t, 'lancelot', email => 'lancelot@example.com');
 $t->get_ok('/tests')->status_is(200)->content_like(qr/is_operator = false;/, "test list rendered without is_operator");
-
-# now the same for scheduled jobs
+note 'now the same for scheduled jobs';
```

Comment:
This note is terribly written and after it we just logout before the next note is printed so it doesn't make much sense.

Reply:
done